CHANGES.rst

```diff
@@ -33,6 +33,7 @@ combine_1d
 
 cube_build
 ----------
+- modified NIRSpec blotting find min and max ra and dec for each slice and only invert those values on slice that fall in range [#5144]
```

Comment:
Is there a verb missing here somewhere, such as "... blotting **TO** find min and max ra and dec ..."?

Reply:
fixed